aiogram/api/types/inline_keyboard_markup.py

```diff
@@ -18,5 +18,49 @@ class InlineKeyboardMarkup(MutableTelegramObject):
     Source: https://core.telegram.org/bots/api#inlinekeyboardmarkup
     """
 
-    inline_keyboard: List[List[InlineKeyboardButton]]
+    row_width: int = 3
+    inline_keyboard: List[List[InlineKeyboardButton]] = []
```

Comment:
Do you know something about mutability in Python?

Reply:
@JrooTJunior it'll work as needed anyway. [Proof/source](https://stackoverflow.com/questions/63793662/how-to-give-a-pydantic-list-field-a-default-value)